scIB/metrics.py

```diff
@@ -747,7 +747,7 @@ def select_hvg(adata, select=True):
         return adata
 
 ### diffusion for connectivites matrix extension
-def diffusion_conn(adata, min_k=50, copy=True, max_iterations=20):
+def diffusion_conn(adata, min_k=50, copy=True, max_iterations=26):
```

Comment:
do we want to change this default, or should we say if no result is reached, then we set the score to 0?

Reply:
But how do we decide how to cut off? I guess fixing a value to cut off here would be pretty random. Doesn't seem to take super long either way

Reply:
doesn't take long? Then okay. Let's keep at 26 then... although the more iterations, the more we just smooth out the data and lose all signal.